modules/custom/az_core/az_core.install

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Contains install and update functions for az_core.
+ */
+
+/**
+ * Install linkit.
+ */
+function az_core_update_920301() {
```

Comment:
If we want to make this a 2.5.x only change, we should change this DB update number (sorry):
```suggestion
function az_core_update_920501() {
```